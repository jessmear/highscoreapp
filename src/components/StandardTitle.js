import React from 'react'

const StandardTitle = props => {
  return (
    <div className='title-box'>
      <h1 className='title'>High Score App</h1>
    </div>
  )
}

export default StandardTitle