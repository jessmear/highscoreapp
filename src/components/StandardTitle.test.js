import { render, screen } from '@testing-library/react';
import StandardTitle from './HighScoreApp';

test('renders star wars title', () => {
  render(<StandardTitle />);
  const title = screen.getByText(/High Score App/i);
  expect(title).toBeInTheDocument();
});


