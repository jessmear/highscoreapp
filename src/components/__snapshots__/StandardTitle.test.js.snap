// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders title should render the Title Component correctly 1`] = `
<div
  className="title-box"
>
  <h1
    className="title"
  >
    High Score App
  </h1>
</div>
`;
